ci/dependencies.sh

```diff
@@ -11,4 +11,8 @@ cargo tree
 cargo tree --duplicate
 cargo tree --duplicate || exit 1
 
+# Check minimal versions.
+cargo update -Zminimal-versions
+cargo tree
```

Comment:
If we add a full check here, then we don't need the checks in other scripts (like `crossbeam-deque.sh`, `crossbeam-epoch.sh`, and so on):

```suggestion
cargo check --all --all-features --bins --examples --tests
```

Reply:
Tests and examples are not published to crate.io as crates, so I think that they are unnecessary. And the changes included in this PR do not allow tests and examples to be compiled with minimal versions.

Reply:
That makes sense, sounds good to me! 👍